spec/decorator_spec.rb

```diff
@@ -0,0 +1,29 @@
+require_relative '../decorator'
```

Comment:
![image](https://github.com/dvcorreia1984/oop-school-library/assets/93039359/94bb12f1-50c7-4b7f-b278-2d57e8e8fe88)

- [ ] Great work but I am not able to run the tests :fearful: Kindly fix the issue and submit your pull request again :+1: